config.gradle

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+project.ext.fluentui_core_versionid='0.0.1'
```

Comment:
can you add here a doc about how versioning would work and when and when not to change this

Reply:
Done. Will be available in next patch